docs/ru/api/createLocalVue.md

```diff
@@ -0,0 +1,27 @@
+## createLocalVue()
+
+- **Возвращает:**
+  - `{Component}`
+
+- **Использование:**
+
+`createLocalVue` возвращает класс Vue, чтобы вы могли добавлять компоненты, примеси и устанавливать плагины без загрузнения глобального класса Vue.
+
+Используйте вместе с `options.localVue`:
+
+```js
+import { createLocalVue, shallow } from '@vue/test-utils'
+import Foo from './Foo.vue'
```

Comment:
shallow -> shallowMount

Reply:
@38elements  thank you! I have not everywhere before made a replacement, now thanks to you, I replaced everywhere with `shallow` on `shallowMount`